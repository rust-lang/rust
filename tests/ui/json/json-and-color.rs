//@ compile-flags: --json=artifacts --error-format=json --color never

fn main() {}
