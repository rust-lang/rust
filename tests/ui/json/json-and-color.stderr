error: cannot specify the `--color` option with `--json`

